test/testSamples/light/SimpleLight.usda

```diff
@@ -8,45 +8,67 @@ def Xform "lights"
 {
     def SphereLight "spotLight"
     {
+        float inputs:intensity = 20000
+        color3f inputs:shadow:color = (1, 0, 0)
+        bool inputs:shadow:enable = 1
+        float inputs:shaping:cone:angle = 50
+        float inputs:shaping:cone:softness = 0.1
+        float inputs:shaping:focus = 0.5
         double3 xformOp:translate = (0, 1, 0)
         uniform token[] xformOpOrder = ["xformOp:translate"]
-
-		float inputs:intensity = 20000
-		bool inputs:shadow:enable = true
-		color3f inputs:shadow:color = (1, 0, 0)
-		
-		float inputs:shaping:focus = 0.5
-		float inputs:shaping:cone:angle = 50
-		float inputs:shaping:cone:softness = 0.1
     }
-	
+
     def SphereLight "pointLight"
     {
-		float inputs:intensity = 200
-		bool inputs:shadow:enable = false
-		color3f inputs:shadow:color = (1, 1, 0)
+        float inputs:intensity = 200
+        float inputs:radius = 0
+        color3f inputs:shadow:color = (1, 1, 0)
+        bool inputs:shadow:enable = 0
+        bool treatAsPoint = 1
```

Comment:
There was a bug here before. We should mark "treatAsPoint" true if we want to make it a poinLight